app/models/gsa18f/procurement.rb

```diff
@@ -0,0 +1,111 @@
+module Gsa18f
+  # Make sure all table names use 'gsa18f_XXX'
+  def self.table_name_prefix
+    'gsa18f_'
+  end
+
+  DATA = YAML.load_file("#{Rails.root}/config/data/18f.yaml")
+  
+
+  class Procurement < ActiveRecord::Base
+    URGENCY = DATA['URGENCY']
+    OFFICES = DATA['OFFICES']
+    RECURRENCE = DATA['RECURRENCE']
+
+    # TODO include ProposalDelegate
+
+    has_one :proposal, as: :client_data
+    # TODO remove the dependence
+    has_one :cart, through: :proposal
+
+    after_initialize :set_defaults
+    
+    validates :cost_per_unit, numericality: {
+      greater_than_or_equal_to: 0,
+      less_than_or_equal_to: 3000
+    }
+    validates :quantity, numericality: {
+      greater_than_or_equal_to: 1
+    }
+    validates :product_name_and_description, presence: true
+
+    def set_defaults
+      # self.not_to_exceed ||= false
+      # self.emergency ||= false
+    end
+
+    def init_and_save_cart(approver_email, requester)
```

Comment:
Why does the `approver_email` need to be passed in explicitly? Can't this method just call that method directly?